### PR TITLE
Low CPU usage loading animation

### PR DESCRIPTION
**Pull request by cmdcolin**
This is a low CPU loading modification

Could look at actual spinners too but this seems like it is not too bad

Note the removal of textAlign center because the text changing length causes the center to move so it is simply left aligned now

### CODE REVIEW

File: packages/linear-genome-view/src/BasicTrack/components/ServerSideRenderedBlockContent.tsx [hunk 1]
```diff
@@ -36,15 +34,20 @@ function LoadingMessage() {
...
+    const timeout = setTimeout(() => setDots(state => (state + 1) % 4), 400)
+    return () => clearTimeout(timeout)
+  })
+
   return shown ? (
     <div className={classes.loading}>
-      Loading &hellip;
-      <LinearProgress />
+      Loading {new Array(dots).fill('.').join('')}
     </div>
```
Comment (garrettjstevens) [inline packages/linear-genome-view/src/BasicTrack/components/ServerSideRenderedBlockContent.tsx:51]: Might be nice to add some styling

```diff
   return shown ? (
-    <div className={classes.loading}>
+    <Typography color="textSecondary" className={classes.loading}>
       Loading {new Array(dots).fill('.').join('')}
-    </div>
+    </Typography>
   ) : null
 }
```

**Comment by cmdcolin** [comment]
Regarding:  "I've found that it's easy to get into a situation where the block is loading but the loading animation isn't anywhere on the screen. It might be worth it to have some sort of animated background that goes the full width of the block, too."

This may be a general problem that also applies to errors on blocks too

**Comment by rbuels** [comment]
Should probably avoid animations that go the full width of the block, they
are pretty intensive for main-thread CPU time

**Comment by rbuels** [comment]
I meant full *size* of the block.  Full width is good, as long as frame rate is not too high

**Pull request merged**